packages/v/vk-bootstrap/xmake.lua

```diff
@@ -24,7 +26,7 @@ package("vk-bootstrap")
             add_requires("vulkan-headers")
             target("vk-bootstrap")
                 set_kind("static")
-                set_languages("cxx14")
+                ]] .. (package:version():ge("1.3.212") and "set_languages(\"cxx17\")" or "set_languages(\"cxx14\")") .. [[
```

Comment:
if we use git url and select branch as version, `add_requires("xx branch")`

package:version() will return nil, we need handle it too.